### PR TITLE
Fix creation of message_queue objects within cronjob callback for sending of reminders to work

### PR DESCRIPTION
**Pull request by iionly**

### CODE REVIEW

**Comment by juho-jaakkola** [comment]
So the feature doesn't work because non-logged-in user (cron) is not able to access non-public events?

File: models/model.php [hunk 1]
```diff
@@ -2058,12 +2045,14 @@ function event_calendar_queue_reminders() {
...
 	}
+
+	elgg_set_ignore_access($ia);
 }
 
 function event_calendar_repeat_reminder_log($e, $start) {
 	// this simple log just uses annotations on the event
 	// TODO - remove log entries for past events
-	create_annotation($e->guid, 'repeat_reminder_log_item', $start, '', 0, ACCESS_PUBLIC);
+	create_annotation($e->guid, 'repeat_reminder_log_item', $start, '', 0, ACCESS_PRIVATE);
```
Comment (juho-jaakkola) [inline models/model.php:2055]: What does this fix?

Comment (iionly) [inline models/model.php:2055]: These annotations are only for internal use, so I don't think public access is necessary. Creating them with private access  seems to me to fit better.

**Comment by iionly** [comment]
Ignoring access within the cronjob is necessary to be able to process all events (regardless of their access level).

File: models/model.php (hunk 1 above)
Comment (juho-jaakkola) [inline models/model.php:2055]: I'd rather include only the minimal changes to a single commit. This change isn't related to fixing the cron.

**Pull request merged**